app/models/order.py

```diff
@@ -91,6 +91,7 @@ def __init__(self,
                  transaction_id=None,
                  paid_via=None,
                  is_billing_enabled=False,
+                 created_at=datetime.datetime.now(datetime.timezone.utc),
```

Comment:
This is the problem. created_at is static - set to the starting time of the server #6703 

@kushthedude @codedsun

Reply:
Making the PR